flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -0,0 +1,81 @@
+create type type_status as enum ('active','deactived','deleted');
+
+create table if not exists sso_user (
+	user_id int generated always as identity not null,
+	user_name text not null,
+	password text not null,
+	email text,
+	kvp json,
+	date_created timestamp,
+	date_modified timestamp,
+	modified_by uuid DEFAULT gen_random_uuid(),
```

Comment:
Nếu dùng kiểu dữ liệu UUID cần cài extension `uuid-ossp`
```suggestion
	modified_by uuid DEFAULT uuid_generate_v4(),
```